DevoxxClientMobile/src/main/java/com/devoxx/views/ExhibitionMapPresenter.java

```diff
@@ -138,15 +147,25 @@ public void setFloor(Floor floor) {
         
         final ETagImageTask imageTask = new ETagImageTask("floor_" + floor.getId(), floor.getImageURL());
         imageTask.setOnSucceeded(e -> {
-            updateImage(imageTask.getValue());
+            final Image image = imageTask.getValue();
+            if (image != null) {
+                updateImage(image);
+            }
+            appBar.setProgressBarVisible(false);
+        });
+        imageTask.setOnFailed(e -> {
+            LOG.log(Level.SEVERE, imageTask.getException().getMessage());
+            appBar.setProgressBarVisible(false);
         });
-        appBar.progressProperty().bind(imageTask.progressProperty());
         new Thread(imageTask).start();
 
         imageTask.image().ifPresent(image -> {
             updateImage(image);
+            appBar.setProgressBarVisible(false);
```

Comment:
This has no effect, because the progress bar will always be made visible again a few lines below.

Reply:
Fixed!